lib/deploy/events/apiGateway/cors.js

```diff
@@ -24,6 +25,16 @@ module.exports = {
         'Access-Control-Allow-Credentials': `'${config.allowCredentials}'`,
       };
 
+      // Enable CORS Max Age usage if set
+      if (_.has(config, 'maxAge')) {
```

Comment:
Seems that maxAge option doesn't work.

Reply:
@horike37 Good catch! maxAge property issue is fixed.

Reply:
cloudformation-template-update-stack.json

<img width="1314" alt="bildschirmfoto 2018-07-25 um 21 05 09" src="https://user-images.githubusercontent.com/611466/43221875-8ac147f8-904e-11e8-85fa-37949ce39799.png">

API Gateway Integration Response

<img width="1144" alt="bildschirmfoto 2018-07-25 um 21 06 37" src="https://user-images.githubusercontent.com/611466/43221953-c56915a2-904e-11e8-8445-0005c1e3574d.png">